src/Akeneo/Tool/Bundle/ElasticsearchBundle/Client.php

```diff
@@ -32,6 +33,9 @@ class Client
     private NativeClient $client;
     private string $idPrefix;
     private int $maxChunkSize;
+    private int $maxExpectedIndexationLatency;
```

Comment:
```suggestion
    private int $maxExpectedIndexationLatencyInMs;
```

Always better to specify the unit in the name, it often avoids many issues.

Reply:
I hesitate to add this comment too :)  
But `Ms` is usually milliseconds, not microseconds

Reply:
I thought it was milliseconds, that's why. hence the importance to specify it.
`maxExpectedIndexationLatencyInMicroseconds`. Long but explicit.

Reply:
i agree with both of you I'd add  we miss type alias to express units, which can be got around with naming conventions...
configuration in units can be precised in configuration. See my last proposal in last commit.

Reply:
I would change the key configuration also: `max_expected_indexation_latency`